sdk/CMakeLists.txt

```diff
@@ -68,24 +66,30 @@ file(GLOB PLATFORM_HEADERS
     ${CHICONY_H}
 )
 
+file(GLOB NETWORK_SOURCES
+	${CMAKE_CURRENT_SOURCE_DIR}/src/connections/network/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/connections/network/*.c
+)
+file(GLOB NETWORK_HEADERS
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/connections/network/*.h
+)
+
 if ( ${OS_SPECIFIC_DIR} STREQUAL ${TARGET_STRING} )
     list(REMOVE_ITEM SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/local_device.cpp)
 endif()
 
-if ( NOT WITH_NETWORK )
-    list (REMOVE_ITEM SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/connections/network/network.cpp)
-    list (REMOVE_ITEM SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/connections/network/network.h)
+#if ( NOT WITH_NETWORK )
     list (REMOVE_ITEM SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/ethernet_device.cpp)
```

Comment:
This always removes ethernet_device.cpp/h. All builds are failing because of this.